widget/completionentry.go

```diff
@@ -97,6 +97,9 @@ func (c *CompletionEntry) ShowCompletion() {
 // calculate the max size to make the popup to cover everything below the entry
 func (c *CompletionEntry) maxSize() fyne.Size {
 	cnv := fyne.CurrentApp().Driver().CanvasForObject(c)
+	if cnv == nil {
+		return fyne.NewSize(0, 0)
+	}
```

Comment:
Looking into entry and select_entry, it does indeed look like you may wish to check for it being nil. I do however wonder if this function is absolutely necessary? Look at the bottom of select_entry.go in the regular fyne widgets and you'll see that we can just set the height of the dropdown to be MinSize and fyne should crop it automatically at the end of the canvas. Maybe we can solve this issue by refactoring the code to avoid this function entirely instead? :)